packages/components/src/templates/next/components/complex/Hero/HeroSearchbar/HeroSearchbar.tsx

```diff
@@ -0,0 +1,39 @@
+import type { HeroSearchbarProps } from "~/interfaces/complex/Hero"
+import { isExternalUrl } from "~/utils"
+import { SearchbarContent } from "./SearchbarContent"
+
+export const HeroSearchbar = (props: HeroSearchbarProps) => {
+  const { backgroundUrl, site } = props
+
+  if (backgroundUrl) {
+    const backgroundSrc =
+      isExternalUrl(backgroundUrl) || site.assetsBaseUrl === undefined
+        ? backgroundUrl
+        : `${site.assetsBaseUrl}${backgroundUrl}`
+    return (
+      <section
+        className="relative flex w-full flex-col justify-center text-base-content-inverse before:absolute before:inset-0 before:bg-[#182236] before:opacity-80 md:min-h-80 lg:min-h-96"
+        style={{
+          backgroundImage: `url('${backgroundSrc}')`,
+          backgroundSize: "cover",
+          backgroundPosition: "center",
+          backgroundRepeat: "no-repeat",
```

Comment:
minor nit: up to you but these 3 seems like can still use the tailwind classes like `bg-cover bg-center bg-no-repeat`. Understand for the backgroundImage we can't so we can leave as that.